src/app/(proper_react)/(redesign)/(public)/how-it-works/page.tsx

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { headers } from "next/headers";
+import { View } from "./HowItWorksView";
+import { getCountryCode } from "../../../../functions/server/getCountryCode";
+import { redirect } from "next/navigation";
+
+export default function Page() {
+  const headersList = headers();
+  const countryCode = getCountryCode(headersList);
+
+  if (countryCode !== "us") {
```

Comment:
I might need to try digging through the flows, but can you confirm that the `countryCode` is always lowercased? I thought we recently uppercased elsewhere for Telemetry or something. But maybe @rhelmer said there was a separate issue to maybe align all of them throughout the code.